framework/SupervisedLearning/ARMA.py

```diff
@@ -318,7 +354,7 @@ def __trainLocal__(self,featureVals,targetVals):
         ## -> instead of a VARMA
         if zeroed.shape[1] == 1:
           # then actually train an ARMA instead
-          zVarma = self._trainARMA(zeroed)
+          zVarma = self._trainARMA(zeroed,mask=None)
```

Comment:
Why is the mask here "None"? If it's an incompatability, we should probably check for that and raise an error to prevent silent failure to pick peaks in this case.

Reply:
If the mask is none this just cause to process the full signal.